src/tests-2.1-transition/pox-transition.ts

```diff
@@ -513,6 +513,34 @@ describe('PoX transition tests', () => {
       test('Verify expected amount of STX are locked', async () => {
         // test stacks-node account RPC balance
         const coreNodeBalance = await client.getAccount(account.stxAddr);
+
+        // todo: FLAKY
+        // full `testAccountBalance` still returned by coreNodeBalance, even though stacking tx was successful
+        expect(BigInt(coreNodeBalance.balance)).toBeLessThan(testAccountBalance);
+        expect(BigInt(coreNodeBalance.locked)).toBe(ustxAmount);
+
+        // test API address endpoint balance
+        const apiBalance = await fetchGet<AddressStxBalanceResponse>(
+          `/extended/v1/address/${account.stxAddr}/stx`
+        );
+        expect(BigInt(apiBalance.balance)).toBeLessThan(testAccountBalance);
+        expect(BigInt(apiBalance.locked)).toBe(ustxAmount);
+
+        // test Rosetta address endpoint balance
+        const rosettaBalance = await getRosettaAccountBalance(account.stxAddr);
+        expect(BigInt(rosettaBalance.account.balances[0].value)).toBeLessThan(testAccountBalance);
+        expect(BigInt(rosettaBalance.locked.balances[0].value)).toBe(ustxAmount);
+      });
+
+      test('REPEAT - Verify expected amount of STX are locked', async () => {
```

Comment:
If we want, we can keep this "repeat" in the test, to debug any future flaky failures. The flake hasn't reappeared in a while though.